api/genomics/scripts/ngs_cohort_extract.py

```diff
@@ -0,0 +1,281 @@
+# -*- coding: utf-8 -*-
```

Comment:
I believe this was copied from the varstore codebase. Can you please leave a comment pointing the original source, and describe any modifications you might have made to this?

Reply:
Added the source and made a PR against GATK for the one change I made. https://github.com/broadinstitute/gatk/pull/7050/files